src/components/common/RegAddEditOrgPerson.vue

```diff
@@ -21,8 +21,22 @@
             class="add-org-header"
             :class="{'error-text': !addPersonOrgFormValid}"
           >
-            <span v-if="isNaN(activeIndex)">Add Business or Corporation</span>
-            <span v-else>Edit Business or Corporation</span>
+            <span v-if="isNaN(activeIndex)">
```

Comment:
Instead of having a span wrapping other spans (and not adding styles or anything), use the Vue `template` element here.

Reply:
LGTM